config/fund_loader_config/FAB/cof_25_eoi.py

```diff
@@ -0,0 +1,151 @@
+LOADER_CONFIG = {
+    "sections_config": [
+        {
+            "section_name": {
+                "en": "1. Expression of interest",
+                "cy": "1. Mynegi diddordeb",
+            },
+            "tree_path": "1037.1.1",
+            "requires_feedback": True,
+        },
+        {
+            "section_name": {
+                "en": "1.1 Organisation details",
+                "cy": "1.1 Manylion y sefydliad",
+            },
+            "tree_path": "1037.1.1.1",
+            "form_name_json": {
+                "en": "organisation-details-25",
+                "cy": "manylion-y-sefydliad",
```

Comment:
Don't we need to append welsh forms with -25 as well?

Reply:
Good spot thank you!